engine/engine.go

```diff
@@ -380,6 +383,34 @@ func (e *Engine) FetchData(ctx context.Context, target string) (_ plugin.Data, d
 	return loadedData.FetchData(ctx)
 }
 
+func (e *Engine) loadEnv(ctx context.Context) plugin.MapData {
+	requiredPrefix := "FABRIC_"
+	if e.config != nil && e.config.EnvVarsPrefix != nil {
+		requiredPrefix = *e.config.EnvVarsPrefix
```

Comment:
Would an empty string pass this and expose all variables?

Reply:
That's the behavior we were thinking about. An alternative would be to have a control value, like `*` that bypasses prefix check and exposes all variables.

Reply:
Hmm, maybe being explicit about this with `*`.
You would expect `nil` and `""` to have similar default behaviour.

Reply:
Yeah, after thinking about it a bit more, I agree. @Andrew-Morozko what do you think? I think being explicit here with `*` == all env vars, is better than ""

Reply:
Yep, but then I think that this field should be a glob value, and the default should be "FABRIC_*", not just "FABRIC_"

Reply:
Actually... Explicitly passing null or "" looks like "do *not* expose anything", not "expose with default prefix". I'll fix this in a next PR, would require us to switch away from simple gojq parsing for global config

Reply:
sure, making a glob seems like a nice and complete solution!

Reply:
Done!